ebu_tt_live/scripts/test/test_scripts.py

```diff
@@ -0,0 +1,12 @@
+
+from subprocess import Popen, PIPE
+from unittest import TestCase
+from ebu_tt_live.scripts import ebu_dummy_encoder
+
+
+class TestDummyScript(TestCase):
+
+    def test_simple_run(self):
+        process = Popen('ebu-dummy-encoder', stderr=PIPE, stdout=PIPE)
```

Comment:
How does this work if the system firewall needs confirmation that the process should accept incoming connections?